cms/settings/local.py.example

```diff
@@ -2,9 +2,6 @@
 # uncomment to use memory cache rather than the Redis service (from docker compose)
 # CACHES["default"]["BACKEND"] = "django.core.cache.backends.locmem.LocMemCache"
 
-# Django Debug Toolbar
```

Comment:
I would still leave this as an option, should one want to disable the debug toolbar locally. thiks means `dev.py` needs to retain `SHOW_TOOLBAR = True # Override in `local.py`` and `"SHOW_TOOLBAR_CALLBACK": lambda x: SHOW_TOOLBAR`,